python/ray/ray_constants.py

```diff
@@ -356,3 +356,6 @@ def to_memory_units(memory_bytes, round_up):
 KV_NAMESPACE_FUNCTION_TABLE = b"fun"
 
 LANGUAGE_WORKER_TYPES = ["python", "java", "cpp"]
+
+# ray whitelist config file absolute path.
+RAY_WHITELIST_PATH = os.environ.get("RAY_WHITELIST_PATH", None)
```

Comment:
```suggestion
RAY_TASK_WHITELIST_FILE = os.environ.get("RAY_TASK_WHITELIST_FILE", None)
```
How about this name?